test/Microsoft.Data.Sqlite.Tests/SqliteConnectionTest.cs

```diff
@@ -175,6 +155,29 @@ public void Open_works()
         }
 
         [Fact]
+        public void Can_use_named_shareable_inMemory_databases()
```

Comment:
Yep, you can always "drop down" to using URIs. This just shows the "supported" ADO.NETty pattern.